robot/Cumulus/tests/browser/relationships/create_relationships.robot

```diff
@@ -1,48 +1,69 @@
 *** Settings ***
 
 Resource        robot/Cumulus/resources/NPSP.robot
-Suite Setup     Open Test Browser
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+...             robot/Cumulus/resources/RelationshipPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
-Test Teardown   Capture Screenshot on Failure
+
+***Keywords***
+
+Capture Screenshot on Failure
+    Run Keyword If Test Failed  Capture Page Screenshot
+
+Setup Test Data
+    &{contact1} =                        API Create Contact    Email=automation@example.com
+    Store Session Record                 Account               &{contact1}[AccountId]
+    Set suite variable                   &{contact1}
+    &{contact2} =                        API Create Contact
+    Store Session Record                 Account               &{contact2}[AccountId]
+    Set suite variable                   &{contact2}
 
 *** Test Cases ***
 
 Create Relationships for contacts
-    [tags]  unstable
-    &{contact1} =  API Create Contact    Email=skristem@robot.com
-    Store Session Record    Account    &{contact1}[AccountId]
-    Go To Record Home  &{contact1}[AccountId]
-    
-    #2 Create a new contact under HouseHold Validation
-    ${contact_id2} =  New Contact for HouseHold
-    &{contact2} =  Salesforce Get  Contact  ${contact_id2}
-    Store Session Record    Account    &{contact2}[AccountId]
-    Page Should Contain  &{contact1}[LastName] and &{contact2}[LastName] Household
-    Select Tab  Related
-    Click Related List Button  Relationships    New
-    Populate Lookup Field    Related Contact    &{contact1}[FirstName] &{contact1}[LastName]
-    Select Value From Dropdown   Type              Parent
-    Click Modal Button        Save
-    ${expected result1}        Catenate    &{contact1}[FirstName] &{contact1}[LastName]    is    &{contact2}[FirstName] &{contact2}[LastName]'s    Parent
-    Log to Console    ${expected result1}
-    ${id}    ${status1}    Check Status    &{contact1}[FirstName] &{contact1}[LastName]
-    Should Be Equal As Strings    ${status1}    ${expected result1}
-    Click Link    link=Show more actions
-    Click Link    link=Relationships Viewer
-    Sleep    5 
+    [Documentation]                      Create 2 contacts with API which inturn creates 2 household accounts.
+    ...                                  Open contact2 record and create a new relationship with contact 1.
+    ...                                  Verify the parent-child relationship is correctly established.
```

Comment:
You are also checking a reciprocal relationship as part of this test and also capturing a screenshot of the relationship diagram so please include those details in the documentation

Reply:
Enhanced documentation reflecting the same